SecretSanta/src/SecretSanta.Web/Controllers/GiftController.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using System.Net.Http;
+using SecretSanta.Web.Api;
+
+namespace SecretSanta.Web.Controllers
+{
+    public class GiftController : Controller
+    {
+        IHttpClientFactory ClientFactory { get; }
```

Comment:
You always want to explicitly set your protection modifiers.
```suggestion
        private IHttpClientFactory ClientFactory { get; }
```